packages/gatsby/src/utils/api-node-helpers-docs.js

```diff
@@ -320,3 +320,11 @@ module.exports.pathPrefix = true;
  * @type {string}
  */
 module.exports.basePath = true;
+
+/**
+ * Check if a specific gatsby feature is enabled or disabled.
+ *
+ * @param {string} feature Feature to check.
```

Comment:
Link to a list of features we have? Otherwise people are left alone here on how to figure out which keys to use

Reply:
I'll add it when the first feature gets added. Reason why I didn't included it yet, maybe the image-service will not be part of next release :)